catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/MetacardComparatorTest.java

```diff
@@ -0,0 +1,201 @@
+package ddf.catalog.cache.impl;
+
+import java.net.URI;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.impl.MetacardImpl;
+
+public class MetacardComparatorTest {
+
+    MetacardImpl cachedMetacard;
+
+    MetacardImpl latestMetacard;
+
+    Date createdDate;
+
+    Date effectiveDate;
+
+    Date expireDate;
+
+    Date modDate;
+
+    String cachedMetacardId;
```

Comment:
Could we create them in the `setup` method and pass them as parameters to `setupCachedMetacard` and `setupLatestMetacard`?